.concourse/cap-release/pipeline.template

```diff
@@ -59,13 +53,25 @@ resources:
     region_name: us-west-2
     regexp: kubecf-bundle-v(.*).tgz
 
+
+# Pool resource with kube cluster information
+{{ range $_, $Backend := (ds "BACKEND").backend -}} # for all backends
+- name: {{$Backend}}-pool.kube-hosts
+  type: pool
+  source:
+    uri: git@github.com:SUSE/cf-ci-pools.git
+    private_key: ((github-private-key))
+    branch: {{$Backend}}-kube-hosts
+    pool: {{$Backend}}-kube-hosts
+{{end}}
+
 {{ $pipelineName := .Env.PIPELINE }}
 
-{{- $obtain_ekcp := `
-            # obtain kubeconfig from ekcp
+{{- $obtain_kubeconfig := `
+            # obtain kubeconfig from pool
+            pool_file=${pool_file:-pool.kube-hosts/metadata}
+            cp ${pool_file} catapult/kubeconfig_$CLUSTER_NAME
```

Comment:
It is possible to also obtain the cluster name from the name of the pool file: `pool.kube-hosts/name`.
That way the following jobs can just depend on `input: pool.kube-hosts`.

On the kubecf pipeline:
https://github.com/cloudfoundry-incubator/kubecf/blob/2d6f7c19f4540cb7154bdebe17794e2232e30fab/.concourse/pipeline.yaml.gomplate#L122-L123

and then just keep doing what you are doing on this PR, use `input: pool.kube-hosts` and compute CLUTER_NAME and KUBECFG values.

That way we accept whatever name the cluster has on the pool, and we can remove all hardcoded CLUSTER_NAME: 
https://github.com/cloudfoundry-incubator/kubecf/blob/b3163dbee4b6117365d46420e8c79b1e0c2f763f/.concourse/cap-release/pipeline.template#L201